CHANGELOG.md

```diff
@@ -15,11 +15,13 @@ Changelog
 
 * update all tests using the filter parameter instead of deprecated types, keys, values ([#98])
 * update some default parameter values in swagger UI to slightly more sensible examples ([#113])
+* restructuring of packages and classes within the controller and output packages ([#117])
```

Comment:
to (start to) be consistent, it would make sense to use the imperative form of the verb rather than the _-ing_ form here:

```suggestion
* restructure packages and classes within the controller and output packages ([#117])
```

Using the imperative present is "generally" recommended AFAIK, e.g. see http://www.gnu.org/prep/standards/html_node/Style-of-Change-Logs.html or https://www.kernel.org/doc/html/v4.10/process/submitting-patches.html#describe-your-changes for example.

Reply:
Ah ok, good to know. I was also mostly using the -ing form in comments of the commits. Will focus on avoiding it then from now on 👍